packages/pds/src/api/com/atproto/server/createSession.ts

```diff
@@ -3,57 +3,68 @@ import AppContext from '../../../../context'
 import { softDeleted } from '../../../../db/util'
 import { Server } from '../../../../lexicon'
 import { AuthScope } from '../../../../auth'
+import { MINUTE } from '@atproto/common'
 
 export default function (server: Server, ctx: AppContext) {
-  server.com.atproto.server.createSession(async ({ input }) => {
-    const { password } = input.body
-    const identifier = input.body.identifier.toLowerCase()
-    const authService = ctx.services.auth(ctx.db)
-    const actorService = ctx.services.account(ctx.db)
+  server.com.atproto.server.createSession({
+    rateLimit: {
+      durationMs: 5 * MINUTE,
+      points: 10,
+      calcKey: ({ req, input }) => `${req.ip}-${input.body.identifier}`,
```

Comment:
We'll want to double-check `req.ip` is set correctly, since if it's not it could represent some piece of infra like a load balancer.  It looks like the behavior there depends on an express setting.